scripts/tfw_lib.sh

```diff
@@ -26,23 +26,80 @@ popd > /dev/null
 declare -r TFW_NAME=`basename $0` # program name (comm name in ps)
 declare -r TFW_NETDEV_PATH="/sys/class/net/"
 declare -r TFW_SCRIPTS="$TFW_ROOT/scripts"
+declare -r CPUS_N=$(grep -c processor /proc/cpuinfo)
 
-declare TFW_DEVS=$(ls $TFW_NETDEV_PATH)
+declare TFW_DEVS=$(ifconfig | grep -o '^[a-z0-9]\+')
```

Comment:
There is no sense to play with queues of down interface, so we fetch up interfaces only.